lib/matplotlib/axes/_axes.py

```diff
@@ -4567,10 +4568,10 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
             - If a sequence of values, the values of the lower bound of
               the bins to be used.
 
-        xscale : {'linear', 'log'}, default: 'linear'
+        xscale : {'linear', 'log', 'symlog'}, default: 'linear'
             Use a linear or log10 scale on the horizontal axis.
```

Comment:
```suggestion
        xscale : {'linear', 'log', 'symlog'}, default: 'linear'
            The scale used for the horizontal axis.

            'symlog' creates a `.SymmetricalLogScale`, which has optional
            parameters. If you want to modify them, pass a tuple
            `('symlog', params_dict)` instead of the plain string 'symlog';
            e.g. `hexbin(..., xscale=('symlog', {'base': 2, 'linthres': 3}))`.
```